src/replication_test.c

```diff
@@ -489,6 +495,7 @@ main(int argc, char **argv)
 				check |= 1 << 3;
 				break;
 			case 'P':
+				strncpy(replica_id, optarg, sizeof(replica_id));
```

Comment:
we should use min of the sizes as len..
or just use strcpy as optarg is null terminated